Stubbing out onboarding call guide (#1308)
openjournals/joss: docs/editing.md

## Onboarding a new JOSS editor

All new editors at JOSS have an onboarding call with an Editor-in-Chief. You can use the structure below to make sure you highlight the most important aspects of being an editor.

**Thing to check before the call**

- Have they reviewed or published in JOSS before? If not, you'll need to spend significantly more time explaining how the review process works.
- Check on their research background (e.g., what tracks they might edit most actively in).
- Make sure to send them the [editorial guide](https://joss.readthedocs.io/en/latest/editing.html) to read before the call.

### The onboarding call

**Preamble/introductions**

- Welcome! Thank them for their application to join the team.
- Point out that this isn't an interview. Rather, this is an informational call designed to give the candidate the information they need to make an informed decision about editing at JOSS.
- 90-day trial period/try out. Editor or JOSS editorial board can decide to part ways after that period.

> - No strict term limits. Some editors have been with us for 7+ years, others do 1-2 years. Most important thing is to be proactive with your editing responsibilities.

Reviewer: ```suggestion
- No strict term limits. Some editors have been with us for 7+ years, others do 1-2 years. Most important thing is to be proactive with your editing responsibilities and communicating any issues with the EiCs.
```

- Explain that there are *two* issues per submission – the pre-review issue and the main review issue.

**The pre-review issue**

- The 'meeting room for the paper'. Where author meets editor, and reviewers are identified.
- Note that the EiC may have initiated a scope review. The editor should not start editing until this has completed. Also, editors are able to query the scope (as are reviewers) if they think the EiC should have (but didn't). 
- Walk them through what is happening in the pre-review issue...
- Editor is invited (likely with GitHub mention but also via email invite (`@editorialbot invite @editor as editor`))
- Once editor accepts they start looking for reviewers.

**Finding reviewers**

- Explain that this is one of the more time-intensive aspects of editing.
- Explain where you can look for editors (your own professional network, asking the authors for recommendations, the [reviewers application](https://reviewers.joss.theoj.org/), similar papers identified by Editorialbot, )
- Point out that we have a minimum of two reviewers, but if more than that accept (e.g., 3/4 then take them all – this gives you redundancy if one drops out).

> - Don't invite one reviewer at a time! If you do this, it may take many weeks to find two reviewers who accept. Try 3/4/5 invites simultaneously.

Reviewer: ```suggestion
- Don't invite only one reviewer at a time! If you do this, it may take many weeks to find two reviewers who accept. Try 3/4/5 invites simultaneously.
```

> - Don't invite only one reviewer at a time! If you do this, it may take many weeks to find two reviewers who accept. Try 3/4/5 invites simultaneously.

Reviewer: add something about using suggested language to invite reviewers

**The review**

> - Once both reviewers are assigned, time to get going!

Reviewer: ```suggestion
- Once at least two reviewers are assigned, time to get going!
```

- Encourage reviewers to complete their review in 4-6 weeks.
- Make sure to check in on the review – if reviewers haven't started after ~1-2 weeks, time to remind them.

> - Your role as editor is not to do the review yourself, rather, your job is to ensure that both reviewers give a good review.

Reviewer: ```suggestion
- Your role as editor is not to do the review yourself, rather, your job is to ensure that both reviewers give a good review and to facilitate discussion and consensus on what the author needs to do.
```

- Walk the editor through the various review artifacts: The checklist, comments/questions/discussion between reviewers and author, issues opened on the upstream repository (and cross-linked into the review thread). 
- Point editors to the ['top tips'](editing.html#top-tips-for-joss-editors) section of our docs. Much of what makes an editor successful is regular check-ins on the review, and nudging people if nothing is happening.
- Do *not* let a review go multiple weeks without checking in.

**Wrapping up the review**

> - Once the review is wrapping up, show the candidate the checks that an editor should be doing (reading the paper, suggested edits, asking for an archive etc.).

Reviewer: ```suggestion
- Once the review is wrapping up, show the candidate the checks that an editor should be doing (reading the paper, suggested edits, asking for an archive etc.)
```

> - Show the `recommend-accept` step which is the formal hand-off between editor and editor-in-chief.

Reviewer: add something about using suggested language to get the last steps done
